src/util/json.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import parseJSONLib from 'parse-json';
+
+import {InvalidJSON} from '../errors';
+
+export function parseJSON(jsonContents: Object, jsonFile: string) {
+  try {
+    jsonContents = stripBOM(jsonContents);
+  } catch (error) {
+    throw new InvalidJSON(
+      `Could not read json file at ${jsonFile}: ${error}`);
```

Comment:
This needs test coverage. Actually, you can just move this test into the new `test-util/test.json.js` suite: https://github.com/mozilla/web-ext/blob/master/tests/unit/test-util/test.manifest.js#L53-L69